IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometryResource/Functions/IfcGradient/Expression.txt

```diff
@@ -1,2 +1,40 @@
-(GradientCurve : IfcGradientCurve) : IfcLengthMeasure;
-  RETURN(1);
+(GradientCurve : IfcGradientCurve, Position : IfcCurveMeasureSelect) : IfcLengthMeasure;
+  
+  LOCAL
+    DistanceAlong : IfcLengthMeasure := 0;
+    Elevation : IfcLengthMeasure := 0;
+    SegmentPlacement : IfcAxis2Placement2D := Segment.Placement\IfcAxis2Placement2D;
```

Comment:
I wonder, what `Segment` is at this stage in the function. Is there a local definition missing?